README.md

```diff
@@ -6,19 +6,19 @@ Contributors that are not SNIA members must first agree to the terms of the SNIA
 The Swordfish Basic Web Client can connect to one or more Swordfish services (including the Swordfish emulator), and present in a web UI frame the entire Swordfish hierarchy.  The basic web client also provides basic capabilities to modify configurable Properties (as specified in the schema), as well an providing a basic interface to add or remove elements from the service through a Web UI.
 
 # Prerequisites
-Install Node.js® and npm if they are not already on your machine.
+Install `node.js` and `npm` if they are not already on your machine.
 Verify that you are running at least node 8.x.x and npm 5.x.x by running node -v and npm -v in a terminal/console window. Older versions produce errors, but newer versions are fine.
 
-Run commands.bat file to install angular-cli and express globally.
+Run `commands.bat` file to install angular-cli and express globally.
```

Comment:
What OS is this valid for?   What configuration? Is this still required?  I didn't do this.